docs/packaging-applications/build-servers/jenkins/pipeline.md

```diff
@@ -111,7 +111,8 @@ _**octopusPushBuildInformation** allows you to push package information to an Oc
 | `commentParser` | Yes | Valid values are `GitHub` and `Jira`.  |
 | `overwriteMode` | Yes | Valid values are `FailIfExists`, `OverwriteExisting` or `IgnoreIfExists`. |
 | `gitUrl` | No | The URL of the repository for the package(s). |
-| `gitCommit` | No | The commit ID for the repository. Available via `git checkout`. |
+| `gitBranch` | No | The branch that was checked out in the repository. Available via `git checkout`. |
+| `gitCommit` | No | The commit ID in the repository. Available via `git checkout`. |
```

Comment:
@benPearce1  `The commit ID in the repository` doesn't sound quite right to me... maybe we can come up with something better

Reply:
```suggestion
| `gitCommit` | No | The commit ID of the most recent commit on the branch. Available via `git checkout`. |
```